cijoe/scripts/sysinfo_report.py

```diff
@@ -215,7 +215,7 @@ def get_motherboard_cpu() -> Tuple[int, Tuple[str, str]]:
         cpu = ""
 
         err, state = cijoe.run(
-            f'get-computerinfo | findstr /B /C:"{CS_PROCESSORS}" /B /C:"{CS_MANUFACTURER}" /B /C:"{CS_MODEL}"'
+            f'powershell "get-computerinfo | findstr /B /C:\\"{CS_PROCESSORS}\\" /B /C:\\"{CS_MANUFACTURER}\\" /B /C:\\"{CS_MODEL}\\""'
```

Comment:
This is fine by me, however, would it be nicer if all the command-output was collected and then processed by the script? For example changing this to:

```python
err, state = cijoe.run('powershell "get-computerinfo"')
```

And then extract from ``state.output()`` the data needed? In other words, replacing the pipes with Python logic.

Reply:
You are right, as we are already looping through the output lines, the pipe is actually unnecessary. I was worried about performance because some of the output can be very long, but I added a check to stop the loop when the desired values have been found and did some checks on performance, and it seems unchanged. :)

Reply:
Great! Btw. seems like the CI started complaining?

Reply:
hehe woops, that's on me, I was using the windows machine to test some local changes which made it unable to rebase :)